README.md

```diff
@@ -76,6 +81,13 @@ Swarm previously only supported local volumes, NFS, and a limited set of Docker
 - [Swarm Ports](https://www.bretfisher.com/docker-swarm-firewall-ports/) - List and description of all the ports used by Swarm Mode (and the very old classic Swarm, if you're into that).
 - [Libnetwork Troubleshooting](https://github.com/moby/libnetwork/blob/master/cmd/diagnostic/README.md) - Official Doc on using network diagnostic tools.
 - [Traefik Proxy](https://github.com/traefik/traefik) - A reverse proxy and load balancer that makes deploying HTTP (and more) published services easy. Swarm Mode docs [start here](https://doc.traefik.io/traefik/providers/docker/#docker-swarm-mode).
+- [Caddy Docker Proxy](https://github.com/lucaslorentz/caddy-docker-proxy) - Caddy based reverse proxy with automatic service discovery based on labels.
+
+### Monitoring
+
+- [docker-engine-events-exporter](https://github.com/neuroforgede/docker-engine-events-exporter) - Prometheus Exporter for Docker Engine Events
+- [docker-engine-networks-exporter](https://github.com/neuroforgede/docker-engine-networks-exporter) - Prometheus Exporter for additional network metrics such as usable ips
+- [promswarm](https://github.com/neuroforgede/promswarm) - Modernized version of [Swarmprom](https://github.com/stefanprodan/swarmprom)
```

Comment:
Well, we kinda resurrected it and added some of our takes to it, deployment uses a lot of our tooling (nothelm, docker-sdp) so that its more extendable. The original code and design is from swarmprom, though - hence the nod with the name. I hope it's okay that the repo is that young, I just moved it to a separate repo earlier today.

About your comment: Sure, will remove Swarmprom from RIP.